src/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+export const ErrorFallback = ({ error }: { error: Error }) => {
+  return (
+    <div data-testid="error-container" className="error-container">
```

Comment:
if you look at the other errors view it has this nice sort of red box around it, it'd be nice to reuse that style

![image](https://github.com/user-attachments/assets/49534961-9a76-4954-aa2d-8ae57d750546)

![image](https://github.com/user-attachments/assets/1bf2761f-fe7f-4f64-8275-a303f4945f9d)